src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'portfolio-welcome-step',
+  templateUrl: 'portfolio-welcome-step.component.html',
+  styleUrls: ['portfolio-welcome-step.component.scss']
+})
+export class PortfolioWelcomeStepComponent {
+  @Input() advanceActiveTab: any;
```

Comment:
You could provide a type here indicating this is a function that accepts an integer parameter.

Reply:
I will add the type as "int"

Reply:
Have a look at this: https://www.typescriptlang.org/docs/handbook/basic-types.html